src/DevAdventCalendarCompetition/DevAdventCalendarCompetition/Views/Results/Index.cshtml

```diff
@@ -85,16 +85,16 @@ else
                 <div class="tab-content">
                     <ul class="nav nav-tabs" id="results" role="tablist">
                         <li role="presentation" class="active">
-                            <a href="#week1" aria-controls="week1" role="tab" data-toggle="tab" data-week="1">@string.Format(CultureInfo.InvariantCulture, @Week, 1)</a>
+                            <a href="#week1" aria-controls="week1" role="tab" data-toggle="tab" data-week="1" data-page="1">@string.Format(CultureInfo.InvariantCulture, @Week, 1)</a>
```

Comment:
probably _data-ranking_, not _data-week_

Reply:
it's literally the week number, so I think the name is right in this case